### PR TITLE
Update installing-to-disk.md

### PR DESCRIPTION
**Pull request by ArkaprabhaChakraborty**
#88 Document dependencies of flatcar-install script 

Added names of the necessary binaries required by the flatcar-install script.

### CODE REVIEW

File: docs/installing/bare-metal/installing-to-disk.md [hunk 1]
```diff
@@ -9,6 +9,20 @@ aliases:
...
+lbzip2 or bzip2 
+wget
+coreutils (gnu core utils package)
+udev (for alpine images it's eudev)
+```
+Some optional binaries are:
+```
+gnupg2
+gawk
+```
```
Comment (pothos) [inline docs/installing/bare-metal/installing-to-disk.md:24]:
```suggestion
## Required dependencies
If you want to use the `flatcar-install` script on some other environment than Flatcar Container Linux, ensure that the following binaries are present:
```
bash
lbzip2 or bzip2 
wget
mount, lsblk (often found in the util-linux package)
grep
cp, dd, mkfifo, mkdir, rm, tee (often found in the GNU coreutils package)
udevadm (found in the udev package, or for Alpine, in eudev)
gpg, gpg2 (found in gnupg2)
gawk (often found in the GNU gawk package)
```

Comment (pothos) [reply docs/installing/bare-metal/installing-to-disk.md:24]: I can't have the correct code block start and end sequence in the suggestion itself, the `// ` was just added as a workaround and should not be committed.

Comment (ArkaprabhaChakraborty) [reply docs/installing/bare-metal/installing-to-disk.md:24]: Okay Ill make the changes

File: docs/installing/bare-metal/installing-to-disk.md [hunk 2]
```diff
@@ -9,6 +9,20 @@ aliases:
...
 ---
+## Required Dependencies
+If you want to use the `flatcar-install` script on some other environment than Flatcar Container Linux, ensure that the following binaries are present:
+```
+bash
+lbzip2 or bzip2 
+mount, lsblk  (often found in the util-linux packaage)
+wget
+grep
+cp, dd, mkfifo, mkdir, rm, tee (found in coreutils or gnu core utils package)
```
Comment (pothos) [inline docs/installing/bare-metal/installing-to-disk.md:20]:
```suggestion
cp, dd, mkfifo, mkdir, rm, tee (often found in the GNU coreutils package or as part of busybox)
```
I would write it like that - the coreutils package is normally from the GNU coreutils project, but on Alpine these would be provided as part of busybox

Comment (ArkaprabhaChakraborty) [reply docs/installing/bare-metal/installing-to-disk.md:20]: Okay Ill change that

Comment (ArkaprabhaChakraborty) [reply docs/installing/bare-metal/installing-to-disk.md:20]: yes for alpine its provided as part of busy box

**Review by pothos — APPROVED** [review]
Thanks!

**Pull request merged**